HACKING.md

```diff
@@ -6,6 +6,42 @@
 
 2. Create images as documented below
 
+## Running the operator locally
+
+1. Install LSO via OLM/OperatorHub in GUI
+
+2. Build the operator
+```
+make build-operator
+```
+
+3. Export env variables required by LSO [deployment](https://github.com/openshift/local-storage-operator/blob/master/config/manifests/4.10/local-storage-operator.clusterserviceversion.yaml#L363)
```

Comment:
Can you make this a permalink, so the link doesn't get broken when we move these to 4.11?

Reply:
Good point - changed.